packages/@expo/cli/src/export/exportAssets.ts

```diff
@@ -12,26 +12,54 @@ import { uniqBy } from '../utils/array';
 
 const debug = require('debug')('expo:export:exportAssets') as typeof console.log;
 
+function mapAssetHashToAssetString(asset: Asset, hash: string) {
+  return 'asset_' + hash + ('type' in asset && asset.type ? '.' + asset.type : '');
+}
+
+export function assetPatternsToBeBundled(
+  exp: ExpoConfig & { extra?: { assetPatternsToBeBundled?: string[] } }
```

Comment:
```suggestion
  exp: ExpoConfig & { experimental?: { assetPatternsToBeBundled?: string[] } }
```

Or, we should put it where the old assetBundlePatterns key was IMO. `extra` is also an option but it doesn't quite fit alongside other known extra fields (`expoClient`, `expoGo`, `eas`)

Reply:
Keys in `experimental` have to be defined in our server schema and `config-types`, whereas `extra` can have any key. So I'd prefer to keep this in `extra` until we have decided on a name for the new property.

Reply:
I think that's by design. `experimental` is designed to be a "internal experiment dumping ground for things that might make their way into a top-level field" while `extra` is more a "per-library-ish dumping ground".

If we do keep it in extra, let's put it in something like `extra?.assets?.assetPatternsToBeBundled` to somewhat signify that this is for the expo-asset library?

Reply:
Sounds good, I will use `extra?.updates?.assetPatternsToBeBundled` as this is specifically for `expo-updates`.